packages/block-editor/src/components/block-list/block.js

```diff
@@ -275,32 +296,43 @@ function BlockListBlock( {
 	 *
 	 * @param {KeyboardEvent} event Keydown event.
 	 */
-	const deleteOrInsertAfterWrapper = ( event ) => {
+	const onKeyDown = ( event ) => {
 		const { keyCode, target } = event;
 
-		// These block shortcuts should only trigger if the wrapper of the block is selected
-		// And when it's not a multi-selection to avoid conflicting with RichText/Inputs and multiselection.
-		if (
-			! isSelected ||
-			target !== wrapper.current ||
-			isLocked
-		) {
-			return;
-		}
+		// ENTER/BACKSPACE Shortcuts are only available if the wrapper is focused
+		// and the block is not locked.
+		const canUseShortcuts = (
+			isSelected &&
+				! isLocked &&
```

Comment:
What's the reason we want to prevent switching modes when the template is locked? Isn't it still useful to be able to navigate content, even if the arrangement of said content can't be modified?

Reply:
That boolean is not used for mode switching. So it should still be possible.